partner/partner-management-service/src/main/java/io/mosip/pms/partner/service/impl/MultiPartnerServiceImpl.java

```diff
@@ -122,12 +123,7 @@ public List<PolicyDto> getAllPolicies() {
                             throw new PartnerServiceException(ErrorCode.PARTNER_ID_NOT_EXISTS.getErrorCode(),
                                     ErrorCode.PARTNER_ID_NOT_EXISTS.getErrorMessage());
                         }
-                        String policyGroupName = policyGroupRepository.findPolicyGroupNameById(partner.getPolicyGroupId());
-                        if (Objects.isNull(policyGroupName) || policyGroupName.equals(BLANK_STRING)) {
-                            LOGGER.info("Policy Group Name is null or empty for partner id : " + partner.getId());
-                            throw new PartnerServiceException(ErrorCode.POLICY_GROUP_NOT_EXISTS.getErrorCode(),
-                                    ErrorCode.POLICY_GROUP_NOT_EXISTS.getErrorMessage());
-                        }
+                        PolicyGroup policyGroup = policyGroupRepository.findPolicyGroupById(partner.getPolicyGroupId());
```

Comment:
why have u removed the throw?
infact do not make any changes in getAllPolicies()

Reply:
removed the changes in getAllPolicies()